### ISSUE TITLE
Suggestion for Moobloom

### ISSUE THREAD

**Issue by Hillionliat** [DereC4/derecearthmod#2]
Moobloom are acting like normal cows but they are planting flowers while walking in duration of 25-30 seconds, the flowers they planting are dandelions and sunflowers, but in adventures they also can plant buttercups, while sheared moobloom drops one buttercup and becomes normal cow
Video - https://youtu.be/QcwbCYU3dCE
Ive tried to wait long enough for them to plant sunflowers but this one moobloom was really stubborn, moobloms plants frequently dandelions, sunflowers are much rarer

**Comment by Hillionliat** [comment]
Video - https://youtu.be/QcwbCYU3dCE
Ive tried to wait long enough for them to plant sunflowers but this on moobloom was really stubborn

**Comment by DereC4** [comment]
Will be added, I thought they were planting buttercups though. I will make it a random assortment of yellow plants to plant, could you provide probabilities then? (Out of 100%)

**Comment by Hillionliat** [comment]
What do you mean by probabilities could you refrase that

**Comment by Hillionliat** [comment]
BTW I have an idea I don't know if you like it but what if you give moobloom to give flower soup that gives regeneration effect, you can get flower soup by feeding cow flowers and then using a bowl you know to make moobloom actually useful and give a reason to find it

**Comment by DereC4** [comment]
> What do you mean by probabilities could you rephrase that, they do plant buttercups but only in adventures in build plates they plant dandelions and sunflowers, dandelions are much often and sunflowers are rare, id say dandelions plant 70% while sunflowers 30% you could also make buttercups rare

Would this be good

- Buttercups 5%
- Sunflower 25%
- Dandelions 70%

**Comment by Hillionliat** [comment]
Yes this is perfect

**Comment by DereC4** [comment]
> BTW I have an idea I don't know if you like it but what if you give moobloom to give flower soup that gives regeneration effect, you can get flower soup by feeding cow flowers and then using a bowl you know to make moobloom actually useful and give a reason to find it

That might deviate too much from the core aspect of the game tho; I can add the shearing later too though

**Comment by Hillionliat** [comment]
I just thought since Mooshrooms do that, Mooblooms could do but with a special potion effect

**Comment by DereC4** [comment]
Do they do that in the base game though

**Comment by DereC4** [comment]
Implemented in 1.4.4.0, thanks for your help!

**Comment by Hillionliat** [comment]
Sorry man it doesn't seem like they not only that they are always running in panic like they've been hit also buttercup on mooblooms head is 2D
![2020-11-24_00 38 23](https://user-images.githubusercontent.com/52645174/100023566-dfca2c80-2ded-11eb-80ff-a7e60badb630.png)

**Comment by DereC4** [comment]
I'll look into this tomorrow I'm too tired today

**Comment by DereC4** [comment]
In the meantime could you create a bug report for this (new issue)
